### DISCUSSION TITLE
Watermark

### DISCUSSION THREAD

**Discussion by clickclack69** [GetPublii/Publii#1501] [General]
Hello,
Is there a convenient way to automatically add a watermark to pictures?

**Comment by candidexmedia** [comment]
You might be able to find a watermarking JavaScript library online or something to add to Publii ([example](https://brianium.github.io/watermarkjs/)), or you could find a desktop tool that can bulk watermark photos before youvbring them into Publii.

**Comment by clickclack69** [comment]
Hello,
Thank you very much for your reply!
